### DISCUSSION TITLE
How to add Marker for react-native-maps

### DISCUSSION THREAD

**Discussion by nilpatil7744** [react-native-maps/react-native-maps#4514] [Q&A]
i am getting this error whenever i am trying to run my project in platform.OS === web

Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

<img width="1440" alt="Screenshot 2022-11-04 at 1 08 49 PM" src="https://user-images.githubusercontent.com/77038652/199923020-d8d95efb-f7ef-4515-b572-461e0d3fafa0.png">

i am using the react-native-maps version @1.3.1 in this project.

i am using it in this way in my component

import MapView, {Marker} from 'react-native-maps';

    <MapView
          onRegionChangeComplete={onRegionChange}
          // provider={PROVIDER_GOOGLE}
          style={{ height: mapheight }}
          customMapStyle={mapStyle}
          region={{
            latitude: userLatitude,
            longitude: userLongitude,
            latitudeDelta: 0.015,
            longitudeDelta: 0.0121,
          }}
          scrollEnabled={true}
        >
          <Marker
            coordinate={{
              latitude: userLatitude,
              longitude: userLongitude,
            }}
            title={'test'}
          />
        </MapView>

**Comment by sssajjad007** [comment]
im not sure, but i think react-native-map not supporting web.

**Comment by ShaneGuther** [comment]
Did you ever end up finding a fix for this?  I am facing a similar issue with Markers.
I managed to get it to stop throwing the error by changing the import from

`import MapView, { PROVIDER_GOOGLE, Marker } from "react-native-maps";`

to its own line without brackets.

`import Marker from "react-native-maps";`

However now it isn't displaying correctly.  The Mapview shows correctly but it appears that the marker is showing up as it's own small map based on my dimensions in the corner of the screen.  It's a similar structure to your code however I am working on iOS.
